eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -119,13 +119,14 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
-    - task: PowerShell@2
-      displayName: Prep template pipeline for release
-      condition: and(succeeded(),eq(variables['TestPipeline'],'true'))
-      inputs:
-        pwsh: true
-        workingDirectory: $(Build.SourcesDirectory)
-        filePath: eng/scripts/SetTestPipelineVersion.ps1
+    - ${{if eq(parameters.TestPipeline, 'true')}}:
```

Comment:
we reference `parameters.TestPipeline`, but when I scroll up to the top of archetype-sdk-client, I don't see it.

Reply:
I fixed the issue.